backupstoragelocation.md

```diff
@@ -36,6 +36,11 @@ spec:
     # Required.
     storageAccount: my-backup-storage-account
 
+    # Secret key of the storage account access key for this backup storage location. Key is looked up in $AZURE_CREDENTIALS_FILE.
+    #
+    # Optional.
+    storageSecretKey: my-backup-storage-secret-key
```

Comment:
the value of this config key will be the name of an env var in the file that's created at the bottom of https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure#create-service-principal, correct?  Just trying to think about the best name for this config key.

Reply:
Something like `storageAccountKeyEnvVarName` is probably more descriptive, though a little wordy. Still thinking.

Reply:
I'm leaning towards what I proposed, `storageAccountKeyEnvVarName`, for clarity - @jaygridley what do you think? @ashish-amarnath @nrb any thoughts here?

Reply:
I would go with `storageAccountKeyEnvVar` to stay consistent with constants velero-plugin-for-microsoft-azure/common.go:29